boa3/compiler/codegenerator.py

```diff
@@ -155,6 +159,35 @@ def convert_byte_array(self, array: bytes):
         data = data_len.to_bytes(prefix_len, sys.byteorder) + array
         self.__insert1(code, data)
 
+    def convert_operation(self, op: IOperation):
```

Comment:
Couldn't we have a method called convert_operation inside the Operation itself? So we don't need the IFs below

Reply:
Wait, I read it wrong again, I will make another comment